cql3/functions/functions.hh

```diff
@@ -17,12 +17,17 @@
 #include "bytes_conversion_fcts.hh"
 #include "aggregate_fcts.hh"
 #include "bytes_conversion_fcts.hh"
+#include "cql3/functions/user_function.hh"
+#include "cql3/functions/user_aggregate.hh"
```

Comment:
This creates a loop, no? In any case functions.hh shouldn't include the more specific user_function.hh, it increases the #include load for its users.

Reply:
Is it okay to create forward declaration of `user_function` and `user_aggregate` in functions.hh? Now, I've done it that way and removed those includes.